### PR TITLE
test(ivy): Add bazel flag to control building ViewEngine or Ivy

### PR DESCRIPTION
**Pull request by mhevery**

### CODE REVIEW

File: packages/core/BUILD.bazel [hunk 1]
```diff
@@ -30,3 +33,20 @@ ng_package(
...
+## Controls if Ivy is enabled. (Temporary target until we permanently switch over to Ivy)
+##
+## This file generates `src/ivy_switch.ts` file which reexports symbols for `ViewEngine` or `Ivy.`
+## - append `--define=ivy=false` (default) to `bazel` command to reexport `./ivy_switch_false`
+##   and use `ViewEngine`;
+## - append `--define=ivy=true` to `bazel` command to rexport `./ivy_switch_true` and use `Ivy`;
+##
+## NOTE: `--define=ivy=true` works with any `bazel` command or target across the repo.
+##
+## See: `\\tools\bazel.rc` where `--define=ivy=false` is defined as default.
```
Comment (alexeagle) [inline packages/core/BUILD.bazel]: why backslashes??

**Review by alexeagle — APPROVED** [review]

File: tools/bazel.rc [hunk 2]
```diff
@@ -53,3 +53,10 @@ test --experimental_ui
 ################################
 
 # Bazel flags for CircleCI are in /.circleci/bazel.rc
+
+################################
+# Temporary Settings for Ivy   #
+################################
+# to determine if the compiler used should be Ivy or ViewEngine one can use `--define=ivy=true` on 
+# any bazel target. This is a temporary flag until codebase is permanently switched to ViewEngine.
```
Comment (trotyl) [inline tools/bazel.rc]: until switched to **ViewEngine**?

Comment (IgorMinar) [reply tools/bazel.rc]: "to Ivy"

**Review by IgorMinar — CHANGES_REQUESTED** [review]
please correct the docs

File: tools/bazel.rc (hunk 2 above)
Comment (mhevery) [reply tools/bazel.rc]: fixed

File: packages/core/BUILD.bazel (hunk 1 above)
Comment (mhevery) [reply packages/core/BUILD.bazel]: fixed

**Comment by mhevery** [comment]
http://test/OCL:197204322:BASE:197204345:1526680326103:766c9bdf

**Review by IgorMinar — APPROVED** [review]
please fix the ci issues

**Review by mhevery — COMMENTED** [review]
LGTM

**Comment by mhevery** [comment]
superseded by https://github.com/angular/angular/pull/24056

**Pull request closed without merging**